routes/users.js

```diff
@@ -1,9 +1,100 @@
-const express = require('express');
-const router = express.Router();
+const express = require('express')
+const { User, db } = require('../database/User.js')
+const router = express.Router()
+const mongoose = require('mongoose')
 
-/* GET users listing. */
-router.get('/', (request, response, next) => {
-  response.send('respond with a resource');
-});
 
-module.exports = router;
+router.get('/', ( request, response, next ) => {
+  response.send( 'respond with a resource' )
+})
+
+router.post('/', ( request, response, next ) => {
+  const { username } = request.body
+  const newUser = new User({
```

Comment:
I would prefer to see this written as a promise composition.  As well, this route would not return any data to the client (which means that the request would hang).